examples/sol-contract/src/processor.rs

```diff
@@ -0,0 +1,110 @@
+//! Program instruction processor
+//! Only the program admin can issue the Init instruction.
+//! And anyone can check the loan with the Loan2Value instruction.
+
+use solana_program::account_info::{
+    next_account_info,
+    AccountInfo,
+};
+use solana_program::entrypoint::ProgramResult;
+use solana_program::msg;
+use solana_program::program_error::ProgramError;
+use solana_program::program_pack::{
+    IsInitialized,
+    Pack,
+};
+use solana_program::pubkey::Pubkey;
+
+use borsh::BorshDeserialize;
+use pyth_sdk_solana::load_price_feed_from_account_info;
+
+use crate::instruction::ExampleInstructions;
+use crate::state::AdminConfig;
+
+pub fn process_instruction(
+    _program_id: &Pubkey,
```

Comment:
If you are actually using these values remove the underscore beginning. (This underscore is used for unused variables)